src/index.js

```diff
@@ -21,19 +19,16 @@ module.exports = Preferences;
  * @param {...Array} preferences The preferences set in wich to look for
  * @return {String|Boolean|Number} The resolved value
  */
-function preferencesQuery(name, fallbackValue) {
-  var value = null;
+function preferencesQuery(name, fallbackValue, ...preferences) {
```

Comment:
The preferences name is a bit confusing since it is a preferences set. What do you think of `preferencesSet`?